src/Node_verify.cpp

```diff
@@ -317,10 +318,22 @@ void Node::verify_vdf(std::shared_ptr<const ProofOfTime> proof, const uint32_t c
 			max_iters = std::max(max_iters, segments[i].num_iters);
 		}
 		if(have_sha_ni) {
-			for(uint32_t j = 0; j < num_lanes; ++j)
+			for(uint32_t j = 0; j < num_lanes; j += 2)
 			{
 				const uint32_t i = chunk * batch_size + j;
-				recursive_sha256_ni(point[j].data(), segments[i].num_iters);
+				const uint32_t k = (j + 1 < num_lanes) ? 1 : 0;
+				uint8_t hashx2[32 * 2];
+				max_iters = std::max(segments[i].num_iters, segments[i + k].num_iters);
```

Comment:
For better readability I would suggest to use a local variable for the two `num_iters`, like `const auto num_iters_0 = segments[i].num_iters` and same for `num_iters_1`.

Reply:
Will do. And thanks for tips to an old C/C++ dude :-) Keeping all feedback in mind going forward.

Reply:
Rev03 commit with changes done.